src/routes/AgreementCreateRoute/AgreementCreateRoute.js

```diff
@@ -116,10 +116,12 @@ const AgreementCreateRoute = ({
         const { id: agreementId } = response;
         // Grab id from response and submit a claim ... CRUCIALLY await the response.
         // TODO we need to think about failure cases here.
-        await claim({ resourceId: agreementId, payload: { claims: payload.claimPolicies ?? [] } });
-
+        if ('claimPolicies' in payload) {
```

Comment:
This doesn't look right to me? The `claimPolicies` should never be in the Agreements payload (and is explicitly omitted above?)